libraries/engage/locations/actions/fetchLocations.js

```diff
@@ -8,7 +9,22 @@ import {
 import { SHOPGATE_STOREFRONT_GET_LOCATIONS } from '../constants';
 
 /**
- * @param {Object} params params.
+ * Returns a list of locations based on search options & defaults. Including geographical radial
+ * distance & other merchant configurations
+ * @param {Object} params Function params
+ * @param {string} [params.postalCode] A postal code
```

Comment:
I think the [ ] brackets are only necessary when you set a default in there

Reply:
this bracket type indicates optional parameters. since the default would be always "undefined" (parameter not sent with the request) i'd keep it like this.